librz/analysis/rtti_msvc.c

```diff
@@ -819,7 +819,17 @@ static void recovery_apply_vtable(RzAnalysis *analysis, const char *class_name,
 		return;
 	}
 
+	RVTableContext context;
+	rz_analysis_vtable_begin(analysis, &context);
+	ut64 startAddress = vtable_info->saddr;
+	RVTableInfo *vtablet = rz_analysis_vtable_parse_at(&context, startAddress);
+	ut64 size = 0;
+	if (vtablet) {
+		size = rz_analysis_vtable_info_get_size(&context, vtablet);
+	}
```

Comment:
Same here, only that you currently don't have the `RVTableContext` available, but you can pass it as a parameter from the callers of `recovery_apply_vtable()` where it can be taken from the `RRTTIMSVCAnalContext`.